### PR TITLE
Finish layout and routers - chap 5

### PR DESCRIPTION
**Pull request by duchv2307**

### CODE REVIEW

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  }
+  ul {
+    float: right;
+    list-style: none;
+    li {
+      float: left;
+      margin-left: 15px;
+    }
+  }
+}
```
Comment (dat-hedspi) [inline app/assets/stylesheets/custom.scss]: end file

File: Gemfile [hunk 2]
```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
```
Comment (dat-hedspi) [inline Gemfile]: replace quote by double quote

File: app/views/layouts/_header.html.erb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "sample app", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home",   root_path %></li>
```
Comment (dat-hedspi) [inline app/views/layouts/_header.html.erb]: ident

File: app/views/layouts/_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+        <li><%= link_to "Help",   help_path %></li>
```
Comment (dat-hedspi) [inline app/views/layouts/_header.html.erb]: ident

File: Gemfile [hunk 5]
```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
...
+# Use bootstrap
+gem 'bootstrap-sass', '>= 3.3.6'
```
Comment (hellovietnam93) [inline Gemfile]: using double quote

File: Gemfile [hunk 6]
```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.0.0', '>= 5.0.0.1'
+# Use bootstrap
```
Comment (hellovietnam93) [inline Gemfile]: remove all remark

File: Gemfile (hunk 2 above)
Comment (hellovietnam93) [inline Gemfile]: using double quote for all

File: app/assets/javascripts/users.coffee [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (hellovietnam93) [inline app/assets/javascripts/users.coffee]: remove this file

File: app/views/static_pages/home.html.erb [hunk 8]
```diff
@@ -1,7 +1,13 @@
...
-<h1>Sample App</h1>
-<p>
-  This is the home page for the
-  <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
-  sample application.
-</p>
+<div class="center jumbotron">
+  <h1>Welcome to the Sample App</h1>
+  <h2>
+  	This is the home page for the
```
Comment (dat-hedspi) [inline app/views/static_pages/home.html.erb]: ident

File: app/assets/stylesheets/custom.scss (hunk 1 above)
Comment (hellovietnam93) [inline app/assets/stylesheets/custom.scss]: lack of end this, please add new blank line at the end of file

File: app/assets/stylesheets/users.scss [hunk 9]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (hellovietnam93) [inline app/assets/stylesheets/users.scss]: remove this file

File: app/helpers/users_helper.rb [hunk 10]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (hellovietnam93) [inline app/helpers/users_helper.rb]: remove this file

File: app/views/layouts/_footer.html.erb [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    by <a href="http://www.michaelhartl.com/">Michael Hartl</a>
+  </small>
+  <nav>
+    <ul>
+      <li><%= link_to "About",   about_path %></li>
```
Comment (hellovietnam93) [inline app/views/layouts/_footer.html.erb]: one space after ```,```

File: app/views/static_pages/home.html.erb (hunk 8 above)
Comment (hellovietnam93) [inline app/views/static_pages/home.html.erb]: ident, back on 1 tab = 2 spaces

File: config/routes.rb [hunk 12]
```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
-  get "static_pages/home"
-  get "static_pages/help"
-  get "static_pages/about"
+  get 'users/new'
```
Comment (hellovietnam93) [inline config/routes.rb]: need?

File: app/views/static_pages/home.html.erb [hunk 13]
```diff
@@ -1,7 +1,12 @@
...
+<div class="center jumbotron">
+  <h1>Welcome to the Sample App</h1>
+  <h2>This is the home page for the
+    <a href="http://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+    sample application.
+  </h2>
+  <%= link_to "Sign up now!", signup_path, class: "btn btn-lg btn-primary" %>
+</div>
+<%= link_to image_tag("rails.png", alt: "Rails logo"),
+"http://rubyonrails.org/" %>
```
Comment (hellovietnam93) [inline app/views/static_pages/home.html.erb]: ident, back on one tab

File: config/routes.rb [hunk 14]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  get "static_pages/home"
-  get "static_pages/help"
-  get "static_pages/about"
   root "static_pages#home"
+  get "/help", to: "static_pages#help"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/signup", to: "users#new"
```
Comment (hellovietnam93) [inline config/routes.rb]: where is create?

Comment (duchv2307) [inline config/routes.rb]: Trong Users controller

**Pull request merged**